Makefile

```diff
@@ -14,6 +14,28 @@ ifeq ($(TAG),)
 TAG := v1.29.3-rke2dev$(BUILD_META)
 endif
 
+ARCH=
+ifeq ($(UNAME_M), x86_64)
+	ARCH=amd64
+else ifeq ($(UNAME_M), aarch64)
+	ARCH=arm64
+else 
+	ARCH=$(UNAME_M)
+endif
+
+ifndef TARGET_PLATFORMS
+	ifeq ($(UNAME_M), x86_64)
+		TARGET_PLATFORMS:=linux/amd64
+	else ifeq ($(UNAME_M), aarch64)
+		TARGET_PLATFORMS:=linux/arm64
+	else 
+		TARGET_PLATFORMS:=linux/$(UNAME_M)
+	endif
+endif
+
+REPO ?= rancher
+IMAGE ?= $(REPO)/hardened-kubernetes:$(shell echo $(TAG) | sed -e 's/+/-/g')
```

Comment:
I think the repo var here is referring to the org and not the repo

Reply:
Forgot to implement the change to use `$(IMAGE)` everywhere, removed `$(ORG)` as it isn't necessary anymore.